src/cmake/dependencies.cmake

```diff
@@ -150,19 +150,19 @@ if(BUILD_WITH_BULLET AND NOT USE_SYSTEM_BULLET)
   # enable BUILD_SHARED_LIBS to build as shared.
   if((NOT CORRADE_TARGET_EMSCRIPTEN) AND CMAKE_VERSION VERSION_LESS 3.13)
     set(BUILD_SHARED_LIBS ON CACHE BOOL "" FORCE)
+    # however the whole Habitat is built with -fvisibility=hidden and Bullet
+    # doesn't export any of its symbols and relies on symbols being visible by
+    # default. Which means we have to compile it without hidden visibility.
+    set(_PREV_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+    string(REPLACE "-fvisibility=hidden" "" CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+    add_subdirectory(${DEPS_DIR}/bullet3 EXCLUDE_FROM_ALL)
```

Comment:
wait no, this isn't right :D the `add_subdirectory()` needs to be outside this `if`, plus you forgot half of the comment below

... and then at the end, after `add_subdirectory()` this `if()` again, which resets the CMAKE_CXX_FLAGS back